File: crates/axhub-helpers/tests/fixtures/ast-validate/kotlin/bad.kt
package sample

// 잘못된 SDK 사용 (block 룰 검출 대상).
fun loadPosts(client: Client) {
    val rows = client.table("posts").where(Ops.or(eq("a", 1), eq("b", 2))).list()
    val n = client.table("posts").where(Ops.not(eq("a", 1))).count()
    val page = client.table("posts").after("cursor").list()
    println("$rows $n $page")
}

File: crates/axhub-helpers/tests/fixtures/ast-validate/go/bad.go
package sample

type listOpts struct {
	after  string
	before string
}

// 잘못된 SDK 사용 (block 룰 검출 대상).
func loadPosts(client *Client) {
	rows := client.table("posts").or(eq("a", 1), eq("b", 2)).list()
	n := client.table("posts").not(eq("a", 1)).count()
	page := client.table("posts").listWith(listOpts{after: "cursor"})
	_ = rows
	_ = n
	_ = page
}
